src/test/java/io/github/thebusybiscuit/slimefun4/testing/tests/tags/TestSlimefunTags.java

```diff
@@ -8,6 +8,7 @@
 
 import org.bukkit.Material;
 import org.bukkit.Tag;
+import org.bukkit.entity.Slime;
```

Comment:
ehhhhhhhhhhh.......

Reply:
wtf lol

Reply:
only if you had the workflow :panLove: 
XD